source/_components/tesla.markdown

```diff
@@ -8,7 +8,7 @@ comments: false
 sharing: true
 footer: true
 logo: tesla.png
-ha_category: Hub
+ha_category: Car
```

Comment:
Instead of the component should the platforms be moved to the categories.

Reply:
Please help me understand hw this should be done correctly. How I understand it, the idea is that it should be easy to find a component based on category. 

Categorizing everything as sensor or hub might be correct as per implementation, but this will make it more difficult to find an item in a certain category. Ideally it should be possible to have several tags, but that might be a longer step to implement.

Reply:
Added car category as per comment in discord #documentation

Reply:
Ok, let's do it the same way as for `DIY`.